crates/cast/tests/e2e/script/declare.rs

```diff
@@ -0,0 +1,50 @@
+use crate::helpers::constants::{SCRIPTS_DIR, URL};
```

Comment:
Let's add a test that will declare the same contract twice and do `stderr_matches` the error msg.

Reply:
My idea is to test somehow that the first declare was successful and the contract is declared on the chain.

Reply:
Agreed. This scenario is covered by regular cast tests either way, so we should be good to go. For now, as this is not essential to have it at the moment, let me just make it in different PR, after release - is that ok?